parallel-install/pkg/deployment/overrideinterceptors.go

```diff
@@ -100,10 +106,18 @@ func (i *DomainNameOverrideInterceptor) getDomainName() (domainName string, err
 		return "", err
 	}
 	if domainName != "" {
+		_, err = patchCoreDNS(i.kubeClient, `(.*)\.local\.kyma\.dev`, i.log)
```

Comment:
Could we make a constant for this value?

Reply:
Done